src/lib/y2storage/autoinst_proposal.rb

```diff
@@ -19,42 +19,29 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "yast"
-require "y2storage/storage_manager"
-require "y2storage/disk_analyzer"
-require "y2storage/proposal"
+require "y2storage/proposal_settings"
```

Comment:
It is used here: https://github.com/yast/yast-storage-ng/pull/296/files#diff-493889b4c9485e86ca3002d10d8112ecL187